extension/rikaicontent.ts

```diff
@@ -848,20 +848,6 @@ class RcxContent {
       }
     }
 
-    //
-    if (
```

Comment:
The wordSearch function now returns results for all words in the dictionary except "～", but I noticed that some words (e.g. "＋α") still get blocked at this if-check before wordSearch is ever called. I'm guessing it's there for performance (similar to the 0x3000 char code check in wordSearch). I removed this if-check and it doesn't appear to have any performance problems and it fixed this problem for words like "＋α".

Reply:
This seems good to me. I can see the goal was to maybe not waste lookups if the first character was seemingly not Japanese looking but that excludes various words in the dictionary and is probably not needed anymore.

If a problem comes up later we can solve it more narrowly then with proper documentation.

Reply:
Though it might be good to add an rikaicontent (or e2e) test for a word like ＋α so that it doesn't get broken in the future. What do you think?

(Something like https://github.com/melink14/rikaikun/blob/7023f15c59bca8573f2120b05a3dd5ac033d64a6/extension/test/rikaicontent_test.ts#LL120C20-L120C20 for example but with different text in the test data.)

Reply:
I think that's a great idea. I added an e2e test for non-Japanese characters triggering xsearch similar to https://github.com/melink14/rikaikun/blob/7023f15c59bca8573f2120b05a3dd5ac033d64a6/extension/test/rikaicontent_test.ts#LL120C20-L120C20.